mapeditor/BitmapHandler.cpp

```diff
@@ -81,7 +81,7 @@ namespace BitmapHandler
 		}
 		else
 		{
-			QImage image(pcx + it, width, height, QImage::Format_RGB32);
+			QImage image(pcx + it, width, height, width * 3, QImage::Format_RGB888);
```

Comment:
3 is 3 bytes?

Reply:
Yes, that's byter per line as I understand